### ISSUE TITLE
Better handling when EGI is down

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-web-editor#222]
originally reported here: https://github.com/openEOPlatform/architecture-docs/issues/129#issuecomment-964029416

at the moment I can not log in on editor.openeo.org or editor.openeo.cloud because EGI seems down
 
I think `https://aai.egi.eu/oidc/.well-known/openid-configuration`  is the failing request that breaks the editor  flow

```
$ curl -i https://aai.egi.eu/oidc/.well-known/openid-configuration
HTTP/1.1 502 Bad Gateway
Server: nginx
Date: Tue, 09 Nov 2021 10:42:35 GMT
Content-Type: text/html
Content-Length: 166
Connection: keep-alive

<html>
<head><title>502 Bad Gateway</title></head>
<body bgcolor="white">
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Results are inconsistent: sometimes I get empty popup, sometimes I get popup that disappears immediately and this error message
![Screenshot from 2021-11-09 11-49-31](https://user-images.githubusercontent.com/44946/140911051-06150045-e877-4cce-bde3-f9287443fe9c.png)

**Comment by soxofaan** [comment]
> Results are inconsistent

as far as I've seen, sometimes the https://aai.egi.eu/oidc/.well-known/openid-configuration request works (resulting in empty popup) and sometimes it doesn't (resulting in "network error" message)

eventually the empty popup renders a "502 Bad Gateway" too:
![Screenshot from 2021-11-09 12-04-21](https://user-images.githubusercontent.com/44946/140913260-e924efb8-0c3b-4f42-b3e5-247b7ce1ef49.png)

**Comment by m-mohr** [comment]
I'm not exactly sure what I can improve here.

In the Browser, I don't get more information. I get purely a "Network error" with no additional details. The only thing I might be able to do is to override the "Network error" with a general "Authentication server is not available" instead...

**Comment by soxofaan** [comment]
Indeed, there is probably little to to against the  "Bad Gateway" error in the popup window.

But if you get a failure on the `https://aai.egi.eu/oidc/.well-known/openid-configuration` (or related) requests, you could put a more informative error message like "Authentication server is not available"

**Comment by m-mohr** [comment]
Okay, moving to the js-client because that's where this happens.
